articles/connections/passwordless/regular-web-app-email-code/v8/index.md

```diff
@@ -20,15 +20,25 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
 Then you can trigger the login using the `callbackURL` option to specify the endpoint that will handle the server-side authentication:
 
 ```html
-<script src="${lock_passwordless_url}"></script>
+<script src="${lock_url}"></script>
 <script type="text/javascript">
-  function login(){
-    // Initialize Passwordless Lock instance
-    var lock = new Auth0LockPasswordless('${account.clientId}', '${account.namespace}');
-    // Open Lock in Email Code mode
-    lock.emailcode( {callbackURL: '${account.callback}'} );
+  function login() {
+    var lock = new Auth0LockPasswordless('${account.clientId}', '${account.namespace}', {
+      oidcConformant: true,                    // Forces an OIDC comformant flow
+      allowedConnections: ['email'],           // Should match the Email connection name, it defaults to 'email'     
+      passwordlessMethod: 'code',              // If not specified, defaults to 'code'
+      auth: {
+        redirectUrl: '${account.callback}',    
+        params: {
+          scope: 'openid email'                // Learn about scopes: https://auth0.com/docs/scopes
```

Comment:
the new default is `openid profile email`. should we have scopes in the docs at all?